config.json

```diff
@@ -470,6 +470,15 @@
       ]
     },
     {
+      "slug": "minesweeper",
+      "difficulty": 7,
+      "topics": [
```

Comment:
Hmm...yes, I would skip the more basic ones, but I think Matrices is a good call.